### ISSUE TITLE
iPad kick / remove participants from meeting

### ISSUE THREAD

**Issue by S-Forsti** [jitsi/jitsi-meet#8272]
Dear GitHub community, 

I currently use Jitsi-meet with my students. Usually I'm logged in with two devices (MacBook + iPad). In doing so, I've noticed that the organiser of a meeting is able to kick / remove participants from a meeting on a MacBook (Browser: Safari or Chrome) but not on the iPad using the jitsi-meet app.

It would be nice if this feature could be implemented for the jitsi -meet app as well.

If it's not possible I'll continue to log in with my Macbook as organiser first, to be able to remove unwanted und undiciplined participants.  

Kind regards
S-Forsti

**Comment by saghul** [comment]
You can long-press on a participant thumbnail and the option to kick them will be revealed in a menu.

**Comment by S-Forsti** [comment]
Thanks for your response. Tried it out and were able to reveal the menu. 

FYI if you use KiwiTalk: 
Thereby I discovered another bug which seems to occur in KiwiTalk only. When I kick a participant (s)he is no longer visible on my screen, however, still part of the meeting and reappears as soon as (s)he does any action (like raising his/her hand). 
Hereby it doesn't matter if I try to kick the participant in the Jitsi-meet app, or if I'm using Safari or Chrome. 

Tried the same on www.meet.jit.si where it works to kick a participant. 
I'll write a notice to those of KiwiTalk.
